src/scripts/main.js

```diff
@@ -18,10 +18,53 @@ const food = {
   },
 };
 
+document.body
+  .appendChild(document.createElement('div'))
+  .setAttribute('id', 'tree');
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  element.appendChild(list);
+
+  for (const i in data) {
+    const listElement = document.createElement('li');
+
+    listElement.textContent = i;
+    list.appendChild(listElement);
+
+    if (Object.keys(data[i]).length > 0) {
+      const subList = document.createElement('ul');
+
+      listElement.appendChild(subList);
+
+      for (const j in data[i]) {
+        const subListElement = document.createElement('li');
+
+        subListElement.textContent = j;
+        subList.appendChild(subListElement);
+
+        if (Object.keys(data[i][j]).length > 0) {
+          const subSubList = document.createElement('ul');
+
+          subListElement.appendChild(subSubList);
+
+          for (const k in data[i][j]) {
+            const subSubListElement = document.createElement('li');
+
+            subSubListElement.textContent = k;
+            subSubList.appendChild(subSubListElement);
+          }
+        }
+      }
+    }
+  }
```

Comment:
You can simplify code like this
```suggestion
  const ul = document.createElement('ul');

  for (const key in data) {
    if (data.hasOwnProperty(key)) {
      const li = document.createElement('li');

      li.textContent = key;

      if (Object.keys(data[key]).length > 0) {
        createTree(li, data[key]);
      }

      ul.appendChild(li);
    }
  }
  
  element.appendChild(ul);
```